apps/src/craft/code-connection/craft.js

```diff
@@ -212,6 +212,9 @@ export const executeUserCode = function (client, code) {
     item: function (blockID, name, data) {
       studioApp().highlight(blockID);
       return { 'name': name, 'data': data };
+    },
+    createBlockPos: function (x, y, z, prefix) {
+      return encodeURIComponent(`${prefix}${x} ${prefix}${y} ${prefix}${z}`);
```

Comment:
Will this eventually replace the copy of this function in blocks.js?  If we plan on keeping both copies it would be good to import the canonical version here.

A better option might be to move all the blocks that use `createBlockPos` to call the interpreted version.